pandas/compat/_optional.py

```diff
@@ -36,11 +36,15 @@
 
 
 def _get_version(module: types.ModuleType) -> str:
-    try:
-        version = module.__version__
-    except AttributeError:
+    version = getattr(module, '__version__', None)
+    if version is None:
         # xlrd uses a capitalized attribute name
-        version = module.__VERSION__
+        version = getattr(module, '__VERSION__', None)
```

Comment:
should we not encode the '__version__' (or '__VERSION__') with the list of deps directly? (or have a way to override); in fact we *already* do this in pandas/util/_print_versions. so actually I would update that to use this list of deps (and also to add the version accessor)